src/hooks/useNotes.js

```diff
@@ -0,0 +1,24 @@
+import { useState, useEffect } from 'react';
+import axios from 'axios';
+import { useAuth } from './../context/auth/authContext';
+export function useNotes() {
+  const { user } = useAuth();
+  const [notes, setNotes] = useState([]);
+  useEffect(() => {
+    const getProducts = async () => {
+      try {
+        const response = await axios.get('/api/notes', {
+          headers: {
+            authorization: user.token,
```

Comment:
In LMS the requested URL mentioned for getting notes is :-
```
GET /api/notes[#](https://mockbee.netlify.app/docs/api/apps/notes-app#1-get-apinotes)
Request URL: /api/notes
```
I think you don't need to pass token to it.

Reply:
Oh sorry, my bad, you do need the encoded token.

Reply:
We need the token to get user notes.